core/src/main/resources/error/error-classes.json

```
@@ -511,6 +511,12 @@
     ],
     "sqlState" : "38000"
   },
+  "FAILED_PARSE_EMPTY_STRING" : {
+    "message" : [
+      "Failed to parse an empty string for data type <dataType>."
+    ],
+    "sqlState" : "42P18"
```

```suggestion
    "sqlState" : "42604"
```

We do know the data type. It's the literal we can't handle

---

Thanks for the correction :-)